.github/workflows/workflow.yml

```diff
@@ -134,6 +142,14 @@ jobs:
         terraform_version: ${{ inputs.terraform_version }}
         refresh_on_pr: ${{ inputs.refresh_on_pr }}
 
+    - if: ${{ github.event_name != 'pull_request' && steps.plan.outputs.has_changes == 'true' && inputs.slack_channel != '' }}
+      name: Send Slack notification
+      uses: Brightspace/terraform-workflows/actions/plan/slack@v4
+      with:
+        slack_channel: ${{ inputs.slack_channel }}
+        slack_token: ${{ secrets.D2L_SLACK_TOKEN }}
+        environment: ${{ matrix.environment }}
```

Comment:
Do you think this should get a continue-on-error: true?

Reply:
yeah good idea